src/test/java/org/folio/rest/impl/SynchronizationAPITests.java

```diff
@@ -49,6 +54,7 @@
   private static final String ITEM_CHECKED_OUT_EVENT_TABLE_NAME = "item_checked_out_event";
   private static final String ITEM_DECLARED_LOST_EVENT_TABLE_NAME = "item_declared_lost_event";
   private static final String ITEM_CLAIMED_RETURNED_EVENT_TABLE_NAME = "item_claimed_returned_event";
+  private static final String ITEM_AGED_TO_LOST_EVENT_TABLE_NAME = "item_aged_to_lost_event";
```

Comment:
Should we add cleaning of this table to @Before declared method?

Reply:
good idea, I think We should.
fixed, thanks